src/frontend/.prettierignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Can't comment against the empty gitleaks.tom file ? What is this ? We have a .gitleaks.toml file ? Is this something else ?

Reply:
Yes, this is a utility to format the files.

Reply:
Both really, the `package.json` is in the ui folder